src/sockperf.cpp

```diff
@@ -3035,17 +3017,21 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
                 // IPv6 in brackets
                 mc_src_ip = m[8];
             }
+        } else if (std::regex_match(line_str, m, regexpr_unix)) {
+            type = line_str[0];
+            addr = std::string(m[1]);
         }
 #endif // NEED_REGEX_WORKAROUND
-        else {
-            log_msg("Invalid input in line %s: "
-                    "each line must have the following format: ip:port or type:ip:port or "
-                    "type:ip:port:mc_src_ip. "
-                    "IPv6 addresses must be enclosed in square brackets.",
-                    line);
-            rc = SOCKPERF_ERR_INCORRECT;
-            break;
-        }
+            else {
+                log_msg("Invalid input in line %s: "
+                        "each line must have the following format: ip:port or type:ip:port or "
+                        "type:ip:port:mc_src_ip or type:/PATH. "
+                        "IPv6 addresses must be enclosed in square brackets."
+                        "Unix domain socket must start with '/'",
+                        line);
+                rc = SOCKPERF_ERR_INCORRECT;
+                break;
+            }
```

Comment:
Please shift this code lock one column left to be aligned with the corresponding `if` blocks.